tasks/main.yml

```diff
@@ -1,6 +1,15 @@
 ---
 # tasks file for prometheus-postgres
 
+- name: Ensure {{prometheus_postgres_system_user}} users exists
+  user:
+    name: "{{prometheus_postgres_system_user}}"
+    comment: "prometheus_postgres_system_user"
+    system: yes
+    shell: /bin/false
+    create_home: no
+  when: "prometheus_postgres_system_user != standard_prometheus_postgres_system_user"
```

Comment:
These lines have failed travis:

```
    /home/travis/build/ome/ansible-role-prometheus-postgres/tasks/main.yml
420      8:13      warning  truthy value should be true or false  (truthy)
421      10:18     warning  truthy value should be true or false  (truthy)
422      11:81     error    line too long (85 > 80 characters)  (line-length)
```

Reply:
What do you suggest? Shorter variable names or
```
when:
  - ....
```

Reply:
Interesting. Not sure what will look best. Back to my previous question, if you don't think "standard_prometheus_postgres_system_user" will ever need changing, you could just use `'postgres'`.

Reply:
You are right...